src/client/module.js

```diff
@@ -0,0 +1,41 @@
+import { Map } from 'immutable';
+
+const PATH = 'js-accounts/';
+const LOGIN = `${PATH}LOGIN`;
+const SET_USER = `${PATH}SET_USER`;
+const LOGGING_IN = `${PATH}/LOGGING_IN`;
+
+const initialState = Map({
+  formType: 'login',
+  isLoading: false,
+  user: null,
+  loggingIn: false,
+});
+
+const reducer = (state = initialState, action) => {
+  const nextState = state;
+  switch (action.type) {
+    case LOGIN: {
+      break;
```

Comment:
Good call. The `LOGIN`case can be removed. `SET_USER` still needs to be filled out for reactivity in the ui when the user is logged in.